pom.xml

```diff
@@ -84,6 +78,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>3.1.2</version>
```

Comment:
```suggestion
```

Version is defined in generated parent pom

Reply:
Done.